WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1363,6 +1365,10 @@ private extension ProductFormViewController {
             break
         }
     }
+
+    func resetProductImages() {
```

Comment:
```suggestion
    private func resetProductImages() {
```

Reply:
This method is already inside a private extension so it's not necessary to mark it private again.